Model/Payment.php

```diff
@@ -430,6 +428,18 @@ public function capture(InfoInterface $payment, $amount)
         }
     }
 
+    private function getCaptureAmount($order, $amountInStoreCurrency)
+    {
+        $orderCurrency = $order->getOrderCurrencyCode();
+        if ($order->getStoreCurrencyCode() == $orderCurrency) {
+            return CurrencyUtils::toMinor( $amountInStoreCurrency, $orderCurrency );
+        } else {
+            // Magento passes $amount in store currency but not in order currency - we have to grab amount from invoice
```

Comment:
@daisy1754 Does this work? The invoice should be created after the capture, not before, so we might not have the data needed. We can probably store the currency with payment, from bolt transaction (I suppose we have it there), by adding it to the $paymentData here: https://github.com/BoltApp/bolt-magento2/blob/master/Helper/Order.php#L1590

Reply:
It worked for me when I tested against my server (2.3.2). Invoice existed but did not have ID at this point

Reply:
We already have currency and convertion rate information in order but u would love to avoid doing own convertion because it can lead to rounding error and price mismatch